pkg/vere/main.c

```diff
@@ -1159,9 +1160,13 @@ _cw_eval(c3_i argc, c3_c* argv[])
 {
   c3_i ch_i, lid_i;
   c3_w arg_w;
+  c3_o jam_l = c3n;
+  c3_o khan_l = c3n;
 
   static struct option lop_u[] = {
     { "loom", required_argument, NULL, c3__loom },
+    { "j", no_argument, NULL, 'j' },
```

Comment:
Wondering if we should use a subcommand for this, i.e. `./urbit eval <pier>` akin to `cram`, `dock`, etc. We don't currently use long args with single letters, e.g. `--j`. Perhaps `--gc` is an exception to this rule. `--j` would be easy to mix up with `-j`

Reply:
Oh looks like this _is_ and `eval` subcommand. I don't think the "j" and "k" subargs to the command are documented in the help output though. Should they be?

Reply:
... Looking at other subcommands which accept arguments other that `--loom`, e.g. `_cw_next`, they also don't document their arguments in the help output, so don't worry about that.

I _do_ think `j` and `k` should be given in long from though, i.e. `--jam` `--jamkahn`

Reply:
I changed it to `--jam` and `--jamkhan`.